addons/mail/static/src/components/message/message.xml

```diff
@@ -27,6 +27,7 @@
                                         'o-message-selected': props.isSelected,
                                         'o_Message_partnerImStatusIcon-mobile': env.messaging.device.isMobile,
                                     }"
+                                    hasOpenChat="hasAuthorOpenChat"
```

Comment:
`hasAuthorOpenChat` code looks wrong for disabling open chat on author that are sole correspondent of current chat: it should use `this.threadView.thread`, not `this.message.originThread`...

In theory message could originate from chat and a channel may follow this chat, so in context of channel we should be able to open chat.

Reply:
Test will require a thread view(er) to work if I make that change, but I guess it is indeed more correct.

Reply:
If you really want test (which is better), could wait for message_fetch PR for having thread viewer.

Reply:
I mean I added a test here already, but it's a message test so I'll have to give it the proper manual thread view id. But it's ok I'll just do it like other tests in that file.

Reply:
done